From 5a1a8d2b920006e54d9de971999b443073f50843 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Tue, 16 Oct 2012 03:30:50 -0400
Subject: [PATCH] Disable PAC test

get-pac-test freezes when run from the ebuild, but succeeds when
building manually. Using Xemake/virtualx.eclass doesn't seem to help.
---
 libproxy/test/CMakeLists.txt | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/libproxy/test/CMakeLists.txt b/libproxy/test/CMakeLists.txt
index 8894425..69543a4 100644
--- a/libproxy/test/CMakeLists.txt
+++ b/libproxy/test/CMakeLists.txt
@@ -17,10 +17,3 @@ if(WIN32)
   target_link_libraries(url-encode ws2_32)
 endif()
 add_test(NAME url-encode COMMAND ${CMAKE_CURRENT_BINARY_DIR}/url-encode)
-
-# Get PAC
-if (NOT WIN32)
-add_executable(get-pac-test ${CMAKE_CURRENT_SOURCE_DIR}/get-pac-test.cpp ${PROJECT_SOURCE_DIR}/url.cpp)
-target_link_libraries(get-pac-test pthread)
-add_test(NAME get-pac-test COMMAND ${CMAKE_CURRENT_BINARY_DIR}/get-pac-test)
-endif()
-- 
1.7.12.3

